.mvn/extensions.xml

```diff
@@ -3,6 +3,6 @@
   <extension>
     <groupId>io.takari.polyglot</groupId>
     <artifactId>polyglot-ruby</artifactId>
-    <version>0.4.7</version>
+    <version>0.4.11-SNAPSHOT</version>
```

Comment:
this is due awaiting a release from upstream, or?

Reply:
Correct. Hopefully this week!